modules/data.remote/R/download.thredds.R

```diff
@@ -1,17 +1,27 @@
 #
-##' @title download.thredds.AGB
-##' @name  download.thredds.AGB
+##' @title download.thredds.data
+##' @name  download.thredds.data
 ##' 
-##' @param outdir Where to place output
-##' @param site_ids What locations to download data at? 
+##' @param outdir file location to place output
+##' @param site_info information about the site. i.e. site_id, latitude, longitude 
```

Comment:
site_info is insufficiently documented. Are there arguments other than site_id, latitude, and longitude? Is site_id the one in BETY or something else? are lat/lon decimal degrees or something else? Is this supposed to be in a list, or a vector, or a data frame, or something else? Are multiple sites allowed

Reply:
ahhh. I see what you mean. I'll update this to be more informative